turbopack/crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -761,7 +761,7 @@ pub(crate) async fn analyse_ecmascript_module_internal(
                         ImportedSymbol::Part(part_id) => Some(ModulePart::internal(*part_id)),
                         ImportedSymbol::Exports => Some(ModulePart::exports()),
                     },
-                    Some(TreeShakingMode::ReexportsOnly) => match &r.imported_symbol {
+                    _ => match &r.imported_symbol {
```

Comment:
Does this mean we will generate `ModulePart::evaluation` when tree shaking is disabled?

Reply:
Asking because IIRC there was panic condition related to it